ui/src/components/bedset-splash-components/beds-table.tsx

```diff
@@ -39,42 +38,56 @@ export const BedsTable = (props: Props) => {
   const { addBedToCart, removeBedFromCart, cart } = useBedCart();
 
   const columns = [
-    columnHelper.accessor('genome_alias', {
+    columnHelper.accessor(row => row.genome_alias, {
```

Comment:
Is there a benefit to this?

Reply:
My editor was mad on me that I am using string